tests/generator/test_passthrough.py

```diff
@@ -58,6 +58,83 @@ def test_passthrough_basic(self):
 method=ignore
 '''}, '''[device-netplan.ethernets.NM-87749f1d-334f-40b2-98d4-55db58965f5f]
 match-device=type:ethernet
+managed=1\n\n''')
+
+    def test_passthrough_basic_new_format(self):
```

Comment:
nitpick: `_new_format` is very generic. Maybe better call it `test_passthrough_basic_mapping`.

(Also in other places where "new_format" is used.)